### PR TITLE
fix(servicestage/instance): fix what causes ecs deployment to fail

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
The value type of host ID list is interface list, not string.
Without properties parameter, the componet will fail if we using obs package to deploy component.
Due to servicestage serivce API's problem, the configuration structure cannot be omited.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. using json marshal and unmarshal function to fix the format error.
2. add obs properties parameter.
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: docs/resources/servicestage_component_instance.md [hunk 1]
```diff
@@ -150,7 +199,7 @@ The `refer_resource` block supports:
   + When `type` is set to **cce**, this parameter is mandatory, and need to specify the namespace of the cluster where
   the component is to be deployed, such as **{"namespace": "default"}**.
   + When `type` is set to **ecs**, this parameter is mandatory, and need to specify the hosts where the component is to
-  be deployed, such as **{"hosts":["04d9f887-9860-4029-91d1-7d3102903a69", "04d9f887-9860-4029-91d1-7d3102903a70"]}}**.
+  be deployed, such as **{"hosts":"[\"04d9f887-9860-4029-91d1-7d3102903a69\", \"04d9f887-9860-4029-91d1-7d3102903a70\"]"}**.
```
Comment (ShiChangkuo) [inline docs/resources/servicestage_component_instance.md:202]: the type of `refer_resource.paramater` is Map not String, please fix it.

Comment (Lance52259) [reply docs/resources/servicestage_component_instance.md:202]: done.

**Pull request merged**